src/main/webapp/app/exam/overview/summary/exercises/quiz-exam-summary/quiz-exam-summary.component.ts

```
@@ -112,12 +112,12 @@ export class QuizExamSummaryComponent implements OnChanges {
     }
 
     getScoreForQuizQuestion(quizQuestionId?: number) {
-        if (this.submission && this.submission.submittedAnswers && this.submission.submittedAnswers.length > 0) {
-            const submittedAnswer = this.submission.submittedAnswers.find((answer) => {
+        if (this.submission() && this.submission().submittedAnswers && this.submission().submittedAnswers!.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)